lighthouse-service/event_handler/evaluate_sli_handler.go

```diff
@@ -537,6 +532,45 @@ func (eh *EvaluateSLIHandler) getPreviousEvaluations(e *keptnevents.InternalGetS
 	return evaluationDoneEvents, nil
 }
 
+func (eh *EvaluateSLIHandler) getPreviousTestExecutionResult(e *keptnevents.InternalGetSLIDoneEventData, keptnContext string) (*keptnevents.TestsFinishedEventData, error) {
+	queryString := fmt.Sprintf("http://mongodb-datastore.keptn-datastore:8080/event?type=%s&project=%s&stage=%s&service=%s&keptnContext=%s&pageSize=%d",
```

Comment:
Is there no environment variable for the URL (`mongodb-datastore.keptn-datastore...`) available that we can access?

Reply:
there is now